--- usbmount.old	2017-11-07 12:01:00.000000000 +0000
+++ usbmount	2022-09-27 22:39:33.473396169 +0100
@@ -106,7 +106,7 @@
 	log info "executing command: mount $DEVNAME"
 	mount $DEVNAME || log err "mount by DEVNAME with $DEVNAME wasn't successful; return code $?"
 
-    elif grep -q "^[[:blank:]]*UUID=\"?$UUID\"?" /etc/fstab; then
+    elif egrep "^[[:blank:]]*UUID=\"?${UUID}\"?[[:blank:]]" /etc/fstab; then
         log info "executing command: mount -U $UUID"
 	mount -U $UUID || log err "mount by UUID with $UUID wasn't successful; return code $?"
 
